proposals/governance/20210915T1500Z.md

```diff
@@ -0,0 +1,89 @@
+# **Direct Integration of Internet Computer with Bitcoin network**
+
+## **Summary**
+
+The Internet Computer will add smart contracts to Bitcoin through an application of Chain Key cryptography that will directly integrate the networks. Smart contracts on the Internet Computer will be able to hold, send and receive bitcoin, without the need for private keys.
+
+
+To vote YES on this proposal means that you agree with the [designed proposed here](https://forum.dfinity.org/t/direct-integration-with-bitcoin/6147/26?u=diegop).
+
+If this NNS proposal passes, then there will be subsequent NNS proposals that will follow the design proposed and will have code updates.
+
+If this NNS proposal fails, then this design will not be implemented and the community will go back to the drawing board.
+
+## **Key people involved**
+
+Thomas Locher, Dieter Sommer, Victor Shoup
```

Comment:
Victor is more active in the ECDSA work. Since I assume you don't want to list 10 people, I'd omit him here

Reply:
done!